modules/interactive_music/doc_classes/AudioStreamPlaybackInteractive.xml

```diff
@@ -9,6 +9,19 @@
 	<tutorials>
 	</tutorials>
 	<methods>
+		<method name="get_current_clip_index">
+			<return type="int" />
+			<description>
+				Return the index of the currently playing clip.
+				You can use this to get the name of the currently playing clip with [method AudioStreamInteractive.get_clip_name].
+				[b]Example usage:[/b]
+				[codeblocks]
+				[gdscript]
+				# Called from inside a AudioStreamPlayer with an active AudioStreamInteractive
+				var playing_clip_name = stream.get_clip_name(get_stream_playback().get_current_clip_index())
+				[/gdscript]
```

Comment:
```suggestion
				[/gdscript]
				[/codeblocks]
```

Reply:
Sorry about that. I didn't realized I missed that one.
Should be fixed now.
Thanks!